src/Utils/Logger.ts

```diff
@@ -18,6 +18,8 @@ import * as vscode from 'vscode';
 
 type MsgList = (number|boolean|string|object)[];
 
+const isDebugMode = process.env.VSCODE_DEBUG_MODE === 'true';
```

Comment:
IHMO, what happens if `process.env.VSCODE_DEBUG_MODE` is undefined? If something bad happens, the code could be
```ts
const isDebugMode = (process.env.VSCODE_DEBUG_MODE !== undefined) && (process.env.VSCODE_DEBUG_MODE === 'true');
```

Reply:
The following code works well without compilation or runtime error. :-)

```ts
let a = undefined;
console.log(a !== 'true');  // this prints true
```

Reply:
👍

Reply:
LGTM!